nautobot/users/filters.py

```diff
@@ -69,6 +69,17 @@ def search(self, queryset, name, value):
         )
 
 
+class TokenFilterSet(BaseFilterSet):
+    q = django_filters.CharFilter(
+        method="search",
+        label="Search",
+    )
+
+    class Meta:
+        model = Token
+        fields = ["id", "key", "write_enabled", "created", "expires"]
```

Comment:
I don't know that it makes much sense to filter Tokens by their `key` value?

Reply:
You have your key but want to look up it's expiration or write_enabled status?